code/script.js

```diff
@@ -316,10 +343,24 @@ const checkMyGuess = (personToCheck) => {
   // 2. Set a Message to show in the win or lose section accordingly
   // 3. Show the win or lose section
   // 4. Hide the game board
+
+  let messgage =""
+  if (secret.name ===personToCheck){
+    alert("Holy Moly you guessed correctly!")
+  } else {
+    alert ("Woopsy daisy better luck next time")
+  }
+  winOrLose.innerHTML=message
+  boardWrapper.classList.add("board-wrapper-inactive")
+  winOrLose.classList.add("win-or-lose-wrapper-active")
 }
 
 // Invokes the start function when website is loaded
-start()
+start(selectQuestion())
```

Comment:
Is it necessary to call upon selectQuestion within the start function? It is allready in the start function to begin with?